### PR TITLE
Run formatting

### PR DESCRIPTION
**Pull request by JElgar**
# Description

Run formatting and enforce in ci

Fixes #<issue_number> or taiga link

# Checklist:

## Creator

- [ ] The target branch is main
- [ ] I have updated the unreleased section of the change log
- [ ] I have reviewed the 'files changed' tab on github to ensure all changes are as expected
- [ ] I have added someone to review the pr

## Reviewer

- [ ] I have verified the above are all completed
- [ ] I have run the code locally to ensure it fufills the requirements of the issue
- [ ] I have reviewed the 'files changed' and commented on any sections which I think are not needed, incorrect or could be improved

## After pull

- [ ] If appropriate I have closed the issue/ moved the trello card

### CODE REVIEW

File: lib/generated/intl/messages_en.dart [hunk 1]
```diff
@@ -13,9 +13,10 @@
 import 'package:intl/intl.dart';
 import 'package:intl/message_lookup_by_library.dart';
 
-final messages = new MessageLookup();
+final messages = MessageLookup();
 
-typedef String MessageIfAbsent(String messageStr, List<dynamic> args);
+typedef MessageIfAbsent = String Function(
+    String messageStr, List<dynamic> args);
```
Comment (kkogut-acrobits) [inline lib/generated/intl/messages_en.dart:19]:
```suggestion
    String messageStr, 
    List<dynamic> args,
);
```

**Review by kkogut-acrobits — APPROVED** [review]
Thanks for taking this into account!

File: lib/generated/intl/messages_en.dart (hunk 1 above)
Comment (JElgar) [reply lib/generated/intl/messages_en.dart:19]: Hmm agreed... I think I tried to convince the linter that was a good idea a while ago and the was some spiel about how they don't let you do that. Let me check again.

Comment (JElgar) [reply lib/generated/intl/messages_en.dart:19]: Ah ok, thats apparently not part of dart format but part of dart fix... Not sure why and there also this open issue about something? https://github.com/dart-lang/dart_style/issues/1253...

**Review by kackogut — APPROVED** [review]

**Pull request merged**